sonar-kotlin-plugin/src/test/java/org/sonarsource/kotlin/api/FunMatcherTest.kt

```diff
@@ -141,6 +141,65 @@ class FunMatcherTest {
         assertThat(funMatcher.matches(ktCallExpression2, BindingContext.EMPTY)).isFalse
     }
 
+    @Test
+    fun `Match method by type and name regex`() {
+        val funMatcher = FunMatcher {
```

Comment:
You can actually use `FunMatcher(nameRegex = "...")`

And I think the test case with just `nameRegex` is missing